Bottom: 510600491f36f98b2ac34c84c7a723fa6d36fc27
Top:    aa70b74182a466ede81dc8830dd03e9b65cf43a0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-24 00:48:43 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3362da6..6fa8575 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -324,13 +324,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
